packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -165,21 +165,25 @@ function useSubMenuState(
 ) {
   const [expandedMenuItemKey, setExpandedMenuItemKey] = React.useState<string>();
   const [submenuTarget, setSubmenuTarget] = React.useState<HTMLElement>();
+  /** True if the menu was expanded by mouse click OR hover (as opposed to by keyboard) */
+  const [expandedByMouseClick, setExpandedByMouseClick] = React.useState<boolean>();
```

Comment:
I think this should be renamed to `[shouldFocusOnContainer, setShouldFocusOnContainer]` instead of its naming now given how it can now mean that the prop was passed in, and not necessarily that it was opened via the mouse. If there is a conflict with the `shouldFocusOnContainer` prop itself we can either access that prop everywhere as `props.shouldFocusOnContainer` or rename one of the two to something else.

```suggestion
  const [shouldFocusOnContainer, setShouldFocusOnContainer] = React.useState<boolean>();
```